src/EnergyPlus/EconomicTariff.cc

```diff
@@ -157,15 +149,13 @@ void GetInputEconomicsTariff(EnergyPlusData &state, bool &ErrorsFound) // true i
     OutputProcessor::Unit UnitsVar(OutputProcessor::Unit::None); // Units sting, may be blank
     Array1D_string NamesOfKeys;                                  // Specific key name
     Array1D_int IndexesForKeyVar;                                // Array index
-    int jFld;
-    std::string CurrentModuleObject; // for ease in renaming.
 
     auto &tariff(state.dataEconTariff->tariff);
 
-    CurrentModuleObject = "UtilityCost:Tariff";
+    std::string_view CurrentModuleObject = "UtilityCost:Tariff";
```

Comment:
This was a pain since the compiler forced me to format all the ShowWarningError strings.

Reply:
Sorry man. Someone was going to have to do that eventually. `string + string + string + ...` is cute and all but `format` is *much* more efficient.